lib/cdo/poste.rb

```diff
@@ -256,7 +256,6 @@ def render_html(bound, locals={}, tracking_id=nil, encrypted_id=nil)
       # Parse the html into a DOM and then re-serialize back to html text in case we were depending on that
       # logic in the click tracking method to clean up or canonicalize the HTML.
       html = Nokogiri::HTML(html).to_html
-      html = inject_litmus_tracking html, tracking_id, encrypted_id
```

Comment:
Do you think the comment above means I can/should remove the Nokogiri line here that "canonicalizes" the HTML as well?

Reply:
That's a good question. It does have the significant side effect of wrapping the template results in a doctype and `<html>` tag, so my assumption is that we need it regardless; but I agree with your interpretation of the comment.

I don't know enough about email to know if document fragments are fine in HTML, but if they aren't then I suspect we still need nokogiri just for that.